Rakefile

```diff
@@ -47,4 +46,18 @@ namespace :docker do
       end
     end
   end
+
+  task :build do
+    ruby_version = ENV['ruby_version'] || '2.6.1'
+    suffix = ENV["image_name_suffix"]
+    unless File.directory?("tmp/ruby")
+      FileUtils.mkdir_p("tmp/ruby")
+      IO.write('tmp/ruby/.keep', '')
+    end
+    # amd64
+    build_image(ruby_version, suffix, nil)
+
+    # arm64
+    build_image(ruby_version, suffix, 'arm64')
```

Comment:
I guess the cause of the build timeout of `master-debug` job is building two images in a single job.
Separating jobs to build images of arm64 architecture presumably resolve the build timeout.